app/src/main/res/layout/completed_story_item.xml

```diff
@@ -69,7 +69,7 @@
           android:maxLines="2"
           android:minLines="2"
           android:text="@{viewModel.completedStoryName}"
-          android:textColor="@color/component_color_completed_progress_activity_completed_story_card_text_color"
+          android:textColor="@color/component_color_shared_content_item_text_view_color"
```

Comment:
Instead of this use `component_color_shared_primary_text_color` color and also in document under `Post PR Changes` mention that we have remove `component_color_shared_content_item_text_view_color `